public/app/features/alerting/unified/home/GettingStarted.tsx

```diff
@@ -266,26 +225,9 @@ function ContentBox({ children, className }: React.PropsWithChildren<{ className
 }
 
 const getContentBoxStyles = (theme: GrafanaTheme2) => ({
```

Comment:
`ContentBox` is very similar in styling to the `@grafana/ui` Card component, but the heading of the Card isn't consistent with how we want to display it here, and switching over in this PR has a few minor knock-on effects/considerations